### ISSUE TITLE
[Typography foundations] Decision log

### ISSUE THREAD

**Issue by sarahill** [Shopify/polaris-react-archive#5988]
# Decision log
Key decisions that impact the direction of the Typography foundations work.

## Text style overrides are out of scope
Typography component overrides (overriding default styles). Decisions on how we handle overrides can impact the whole system and should be considered as a separate project

---

## Type scale adjustments
TL;DR The type scale was adjusted and ranges from 12px to 40px with each increment being multiplied or divided by a ratio of 1.2 to get the next or previous value in the scale.

**Why**
The scale has minor adjustments to lean into the task based UI of the admin. The scale allows for a wider range of headings to create a strong sense of hierarchy.

---

## 3 roles
TL;DR: Text styles are organized by three roles: display, heading and body. Having the three roles provide some clarity and opinion on how to apply text within the admin while still leaving some flexibility to flex these styles when needed.

**Why**
During concept testing, we tested two concepts.
- Concept 1 was organized into two roles: heading and body.
- Concept 2 was organized into five roles: Display, Title, Heading, Body, and Label.

We found that concept 1 provided more perceived flexibility but also added more cognitive load to try and understand which text style to use. Concept 2 was generally easier to pinpoint where to start but the nuances between Display, Title, Heading, and Label made it difficult to know which one to use and what the difference between them was.

Each role has a purpose along with a set of options to allow for flexibility.

---

**Add decision/rationale for:**
- [ ] single vs. multiple text components
- [ ] token naming convention
- [x] type scale
- [x] roles/text styles
